File: Harvard-CS-Course/Serafin-dev-cs50-problems-2021-x-sentimental-mario-more/mario.py
def pyramid(height):
    counter_3 = 1
    counter_2 = 1 
    counter_1 = height - 1
    
    #Rows
    for i in range(height):
       
        #Print Blank spaces from the left
        counter = 0
        j = counter_1
        while counter < j:
    	    counter += 1
    	    print(" ", end="")

        
        counter_1 = counter_1 - 1

        # print the pyramid from the left
        k = 1
        while k <= counter_2: 
            numeral = '#'
            print (numeral, end="")
            k += 1
        
        counter_2 += 1
        
        # Print double blank space in the middle of the pyramid
        print("  ", end="")
        
        # print the pyramid from the right
        k=1
        while k <= counter_3 :
            numeral = '#'
            print (numeral, end="")
            k += 1
        
        counter_3 += 1
        print("")
    

def main():
    # Get Height integer from the user 
    height = input("Height: ")
    height = int(height)
    while height < 1 or height > 8:
        heigth = input("Height: ")
    
    pyramid(height)
    
main()

File: Harvard-CS-Course/Serafin-dev-cs50-problems-2021-x-mario-more/mario.c
#include <stdio.h>
#include <cs50.h>

//personalized function that creates the pyramids
void pyramid(int height)
{   
    
    int counter_3 = 1;
    int counter_2 = 1; 
    int counter_1 = height-1;
    
    //Rows
    for (int i = 0; i<height; i++)
    {   

        //Print Blank spaces from the left
        int j=counter_1;
        int counter = 0;
        while (counter < j)
        {
            char blank =' ';
    	    printf ("%c", blank);
    	    counter++;
        }
        counter_1 = counter_1 - 1;

        //print the pyramid from the left

        for (int k=1; k<=counter_2 ; k++)
        {   
            char numeral = '#';
            printf ("%c", numeral);
        }

        counter_2++;
        //Print double blank space in the middle of the pyramid
        printf("  ");

        //print the pyramid from the right
        for (int k=1; k<=counter_3 ; k++)
        {   
            char numeral = '#';
            printf ("%c", numeral);
        }

        counter_3++;

        printf("\n");
    }
}

int main (void)
{   
   //Get Height integer from the user    
   int height;
   do
   {
        height = get_int("Height: ");
   }
   while (height < 1 || height > 8);

   pyramid(height);
}
